bundles/core/src/main/java/com/adobe/cq/wcm/core/components/models/Page.java

```diff
@@ -238,6 +238,18 @@ default String getTitle() {
         throw new UnsupportedOperationException();
     }
 
+    /**
+     * If this page is associated with a Template, then this method will return the Template's client libraries categories for the head tag.
```

Comment:
• client libraries categories -> client library categories
• "client library categories included in the page head"